### PR TITLE
HOMEWORK-1: homework_4

### PR DESCRIPTION
**Pull request by CherednichenkoNikita**
renamed branch

### CODE REVIEW

File: src/main/java/com/java_professional/cherednichenko/homework/lesson_4/task_2/WriteInteger.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    public static void main(String[] args) {
+        List<String> arrayList = new ArrayList<>();
+        try (FileWriter writer = new FileWriter("Integer.txt", false)) {
+            Random random = new Random();
+            for (int i = 0; i < 10; i++) {
+                writer.write(String.valueOf(random.nextInt()));
+                writer.append('\n');
+            }
+            writer.flush();
```
Comment (ochedenis) [inline src/main/java/com/java_professional/cherednichenko/homework/lesson_4/task_2/WriteInteger.java]: Try with resources automatically closed writer. A close will always also flush. So you don't need flush by hands.

**Review by ochedenis — DISMISSED** [review]

File: src/main/java/com/java_professional/cherednichenko/homework/lesson_4/task_1/Write.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+//Задание 2
+//        Создайте файл, запишите в него произвольные данные и закройте файл.
+//        атем снова откройте этот файл, прочитайте из него данные и выведете их на консоль.
+public class Write {
+    static Logger logger = Logger.getLogger(Write.class.getName());
```
Comment (yuriy-tsarenko) [inline src/main/java/com/java_professional/cherednichenko/homework/lesson_4/task_1/Write.java]:
```suggestion
    static Logger logger = Logger.getLogger(Write.class);
```
Change it

File: src/main/java/com/java_professional/cherednichenko/homework/lesson_4/task_2/WriteInteger.java (hunk 1 above)
Comment (yuriy-tsarenko) [reply src/main/java/com/java_professional/cherednichenko/homework/lesson_4/task_2/WriteInteger.java]: A close will always also flush in FileOutputStream family, but it doesn't work with FileWriter family.

**Review by yuriy-tsarenko — CHANGES_REQUESTED** [review]
Fix code according to comments

**Pull request merged**